BootloaderCommonPkg/Include/Library/BootloaderCommonLib.h

```diff
@@ -64,6 +64,9 @@ typedef struct {
 #define TO_MM_PCI_ADDRESS( Addr ) \
   MM_PCI_ADDRESS ( ((Addr) >> 16) & 0xFF, ((Addr) >> 8) & 0xFF, (Addr) & 0xFF, 0)
 
+#define TO_PCI_LIB_ADDRESS( Addr ) \
```

Comment:
why do we need both TO_MM_PCI_ADDRESS  and TO_PCI_LIB_ADDRESS ?

Reply:
Before MMIO access is available, PCI lib address is required in early phase, e.g. to enable SPI flash access.

Reply:
Since they are the same,  any reason for duplication with different macro names ?

Reply:
They are different. one is for MMIO access, one is for CF8/CFC legacy access.